dendrite/pipeline.yml

```diff
@@ -67,6 +67,40 @@ steps:
           limit: 2
         - exit_status: 2
           limit: 2
+  
+  - label: "SyTest - :go: 1.13 / :postgres: 9.6 / full HTTP APIs"
```

Comment:
Sad noises about 9.6, is *anyone* actually running a 9.6 with dendrite these days?